Controller/ControllerTrait.php

```diff
@@ -12,17 +12,23 @@
 namespace FOS\RestBundle\Controller;
 
 use FOS\RestBundle\View\View;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use FOS\RestBundle\View\ViewHandlerInterface;
 use Symfony\Component\HttpFoundation\Response;
 
 /**
- * Base Controller for Controllers using the View functionality of FOSRestBundle.
+ * Trait for Controllers using the View functionality of FOSRestBundle.
  *
  * @author Benjamin Eberlei <kontakt@beberlei.de>
+ * @author Lukas Kahwe Smith <smith@pooteeweet.org>
  */
-abstract class FOSRestController extends Controller
+trait ControllerTrait
```

Comment:
Oops look too fast... 
If there are two traits, the problem will be the same (moreover ``get`` belongs to ``Controller``) ... 
You can maybe check if the ``get`` method exists on ``$this`` or if ``$this`` is an instance of ``Controller``.